src/components/Legend.vue

```diff
@@ -137,16 +138,29 @@ import { getPartyName } from '@/models/candidates';
 import { Catastrophe, CatastropheType, formatDescription, getIconUrl, getTypeName } from '@/models/catastrophes';
 import { DistrictProperties } from '@/models/map';
 import { useStore, CURRENT_YEAR } from '@/stores/store';
-import { defineComponent } from 'vue';
+import { defineComponent, onMounted, ref } from 'vue';
 import vSelect from 'vue-select';
+import { Collapse } from 'bootstrap';
+
 
 export default defineComponent({
     emits: ['onRequestCatastropheFocus'],
     components: { vSelect },
     setup() {
         const store = useStore();
         const catastropheTypes = Object.values(CatastropheType);
-        return { store, catastropheTypes };
+        const catastrophes = ref<HTMLElement>();
+        onMounted(() => {
```

Comment:
Si tu retournes `catastrophes` dans l'objet du setup, tu vas pouvoir y référer dans `mounted()` avec `this.catastrophes`, plus bas, et t'auras pas besoin d'utiliser `onMounted()` . Rendu là par contre va peut-être falloir que tu le renommes pour que ça soit plus clair (`catastrophesElem`?)

Reply:
Nice, merci! Done.